client/client.go

```diff
@@ -283,20 +283,19 @@ func (c *CqlClientConnection) outgoingLoop() {
 	go func() {
 		abort := false
 		for !abort && !c.IsClosed() {
-			if outgoing, ok := <-c.outgoing; !ok {
-				if !c.IsClosed() {
-					log.Error().Msgf("%v: outgoing frame channel was closed unexpectedly, closing connection", c)
-					abort = true
-				}
-				break
-			} else {
+			select {
+			case outgoing := <-c.outgoing:
 				log.Debug().Msgf("%v: sending outgoing frame: %v", c, outgoing)
 				if c.modernLayout {
 					// TODO write coalescer
 					abort = c.writeSegment(outgoing, c.conn)
 				} else {
 					abort = c.writeFrame(outgoing, c.conn)
 				}
+			case <-c.ctx.Done():
+				log.Error().Msgf("%v: outgoing frame channel was closed unexpectedly, closing connection", c)
```

Comment:
`%v: outgoing frame channel was closed unexpectedly, closing connection` was logged when c is not closed but channel is. the new case is for `c.ctx.Done` which is kind "expected"?. could you update the message to align the new case? (and the log in the other loop below)

also, is setting `abort` to `true` neccssary? assuming `IsClosed` should return `true` now. I guess it's safer to set it.